features/admin/pod.feature

```diff
@@ -69,14 +69,12 @@ Feature: pod related features
     And the pod named "pod-pull-by-tag" status becomes :running
     Given I register clean-up steps:
     """
-    I run the :oadm_manage_node admin command with:
-      | node_name   | <%= cb.pod_node %> |
-      | schedulable | true               |
+    I run the :oadm_uncordon_node admin command with:
+      | node_name | <%= cb.pod_node %> |
```

Comment:
why not use `node schedulable status should be restored after scenario` step instead of this kind of clean-up?

Reply:
@akostadinov Fixed in https://github.com/openshift/verification-tests/pull/413